week-2/mandatory/2-ecommerce-db/task.md

```diff
@@ -35,16 +36,53 @@ Open the file `cyf_ecommerce.sql` in VSCode and examine the SQL code. Take a pie
 Once you understand the database that you are going to work with, solve the following challenge by writing SQL queries using everything you learned about SQL:
 
 1. Retrieve all the customers' names and addresses who live in the United States
+
+    select name, address from customers where country = 'United States';
+
 2. Retrieve all the customers in ascending name sequence
+
+    select * from customers order by name desc;
+
 3. Retrieve all the products whose name contains the word `socks`
+
+    select * from products where product_name like '%socks%';
+
 4. Retrieve all the products which cost more than 100 showing product id, name, unit price and supplier id.
+
+     select p.id, p.product_name, pa.unit_price, pa.supp_id from products p join product_availability pa on p.id = pa.prod_id where pa.unit_price>100;
```

Comment:
Nice work
Generally try to format it in a nicer way
This may help
https://stackoverflow.com/questions/519876/sql-formatting-standards